common/configParser.go

```diff
@@ -163,3 +165,33 @@ func GetSkBytesFromP2pKey(p2pKeyFilename string) ([]byte, error) {
 
 	return skBytes, nil
 }
+
+// CreateP2pKeyPair will create a set of key pair for p2p based on provided pem file. If
+// the provided key is empty it will generate a new one
+func CreateP2pKeyPair(
```

Comment:
unrelated to `common` package. This will need to be moved into the crypto components factory.

Reply:
moved to crypto factory